server/config/environment.go

```diff
@@ -210,6 +211,10 @@ func GetEnvironment() envVars {
 	if environment.APPLICATION_ID == "" {
 		log.Warn("APPLICATION_ID environment variable is either unset or is an empty string, deployment telemetry will not contain the applicationid property")
 	}
+	environment.START_TIME = env.Get("START_TIME")
+	if environment.START_TIME == "" {
```

Comment:
Is there any reason why we can't have a default for start time?

Reply:
Any idea what should we set as default?

Reply:
I would use this for default value: `time.Now().Format(time.RFC3339)`

Reply:
done